pgxn/neon/control_plane_connector.c

```diff
@@ -433,32 +433,30 @@ MergeTable()
 											   entry->name,
 											   HASH_ENTER,
 											   NULL);
+			RoleEntry * old;
+			bool found_old = false;
```

Comment:
I believe the Postgres convention is to put declarations at the top of their scope.

Reply:
Fixed